testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -254,6 +263,16 @@
         };
     }
 
+    @DataProvider
+    public static StorageFormat[] storageFormatsWithZeroByteFile()
+    {
+        return new StorageFormat[] {
```

Comment:
why just those three?

Reply:
should we also test against compressed vs non-compressed?

Reply:
compressed vs non-compressed in parquet/orc or ?

Reply:
Yeah - I meant parquet and orc with some compression enabled. But I am not sure if that is beneficial. You tell me :)

Also why did we pick those three specific formats. Why skipping CSV, SEQUENCEFILE, RCBINARY and others.

Reply:
> But I am not sure if that is beneficial.
I think it won't be helpful. Since the footer is anyway uncompressed and we don't reach till that point in case of empty file. 

Will merge the two commits as it causes a bit confusion. I was planning to have like 
1. Add test coverage.
2. Extend them in next commits. 

It kind of causes a confusion.